src/app/audio/audio.model.ts

```diff
@@ -0,0 +1,13 @@
+import {User} from "../user/user.model";
+import {MapLocation} from "../map-location/map-location.model";
+
+export class Audio {
+  constructor(
+    public id: string,
+    public name: string,
+    public description: string,
```

Comment:
Description chyba można dać optional? Chyba że na backendxie jest @NotNull to musi być tak